test/spec/property-operators/giv/assignment/property-to-value/source.djs

```diff
@@ -0,0 +1 @@
+document giv body is 'blah'
```

Comment:
Maybe we shouldn't assign strings to elements?  Trying this in Firefox actually throws a TypeError

Reply:
well i only set this as the test, but i can update it to an html string? idk what would be more appropriate. these only get compared to the parsed out JS and not actually ran (the js)